frontend/packages/dev-console/src/components/import/image-search/ImageStreamNsDropdown.tsx

```diff
@@ -63,7 +63,14 @@ const ImageStreamNsDropdown: React.FC = () => {
   );
 
   React.useEffect(() => {
+    if (
+      initialValues.imageStream.image &&
+      values.imageStream.image !== initialValues.imageStream.image
+    ) {
+      initialValues.imageStream.image = values.imageStream.image;
```

Comment:
resetting initial values seems odd @rohitkrai03 @divyanshiGupta thoughts?

Reply:
It's generally not recommended to change the initial values but here I think resetting the initial value is fine since we just need the initial value only when the form is opened.

Reply:
This effect gets fired the first time `ImageStreamDropdown` is changed, it's meant to verify the current namespace permissions, but ends up resetting the selected ImageStream name back to the initial value.